uiowa/uiowa
# Clean up frontend dependencies.

This PR removes fractal as a build dependency in combination with https://github.com/uiowa/uids/pull/73 and https://github.com/uiowa/hds/pull/85. It also adds some additional options to `blt frontend` which makes it explicitly a production install. It is possible to install devDependencies by manually running `yarn install` if necessary. I added jquery and popper.js to uiowa_bootstrap which resolves a warning during the install process.

I do not have fractal installed in node_modules after this. I did `rm -rf node_modules` beforehand though. I'm not if everyone will have to do that as well.

We'll want to change the required versions before merging this.

- [x] Update uids and hds versions in uids_base and collegiate_theme, respectively.

### Testing
- Check out branch locally and run `blt frontend`.
- Make sure fractal is not in node_modules (see note above)
- Make sure sites look as expected for uiowa_bootstrap (hr, brand), collegiate_theme and uids_base themes.
